### PR TITLE
Use new cos image in test

### PR DESCRIPTION
**Pull request by yguo0905**
/assign @krzyzacy @yujuhong

### CODE REVIEW

File: experiment/test_config.yaml [hunk 1]
```diff
@@ -760,8 +760,8 @@ images:
     - --gcp-node-image=gci
   cosbeta:
     args:
-    - --image-family=cos-beta
-    - --image-project=cos-cloud
+    - --env=KUBE_GCE_NODE_IMAGE=cos-only-for-testing-63-10032-39-0
+    - --env=KUBE_GCE_NODE_PROJECT=gke-node-images
```
Comment (krzyzacy) [inline experiment/test_config.yaml]: why we need to switch from flags to global envs?

Comment (yguo0905) [reply experiment/test_config.yaml]: `kubernetes_e2e.py` doesn't support `--image` with `--image-project` yet.

Comment (yguo0905) [reply experiment/test_config.yaml]: https://github.com/kubernetes/test-infra/issues/3320

**Comment by yujuhong** [comment]
lgtm

**Pull request merged**